docker-compose.yml

```diff
@@ -19,8 +19,7 @@ services:
       # https://bugs.openjdk.java.net/browse/JDK-8230305
       # cgroups v2 resource reservations only work w/ java 15+ ATM, so it's best for our java processes to be explicit
       # with max memory.
-      - JAVA_TOOL_OPTIONS=-Xmx700m
-      - JAVA_TOOL_OPTIONS=-Dio.deephaven.console.type=${IO_DEEPHAVEN_CONSOLE_TYPE}
+      - JAVA_TOOL_OPTIONS=-Xmx4g -Dio.deephaven.console.type=${IO_DEEPHAVEN_CONSOLE_TYPE}
 #      - JAVA_TOOL_OPTIONS=-Dlogback.configurationFile=logback-debug.xml
 #      - JAVA_TOOL_OPTIONS=-Dlogback.configurationFile=logback-minimal.xml
```

Comment:
Are these commented-out lines misleading if they cannot be uncommented for what otherwise looks like a desired effect? Do you mind prefixing with a hint that is a little clearer that you have to append to the existing line and not just uncomment?

Reply:
I'll add a comment to hopefully make it clearer - you are right, uncommenting these lines will break stuff.